src/features/click-to-load.js

```diff
@@ -499,7 +566,7 @@ function showExtraUnblockIfShortPlaceholder (shadowRoot, placeholder) {
  *   in the document will be replaced instead.
  */
 async function replaceClickToLoadElements (targetElement) {
-    await ready
+    await afterInit
```

Comment:
> It seems like you have only one call site into this method in update, are you able to await there for pageload, init etc rather than scattering awaiting through the code?

We could potentially wait for `afterInit` before calling `replaceClickToLoadElements` like you suggest, though I think having the await inside the function makes more sense personally.

We don't want to wait for page load before replacing elements though, since we want the placeholders to show up right away.